tests/Jellyfin.Server.Implementations.Tests/IO/ManagedFileSystemTests.cs

```diff
@@ -26,7 +28,16 @@ public ManagedFileSystemTests()
             string expectedAbsolutePath)
         {
             var generatedPath = _sut.MakeAbsolutePath(folderPath, filePath);
-            Assert.Equal(expectedAbsolutePath, generatedPath);
+
+            if (MediaBrowser.Common.System.OperatingSystem.Id == OperatingSystemId.Windows)
```

Comment:
It'd be nice if there were a way of specifying a Windows path in the `InlineData` in order to test paths from different platforms. I think a cleaner way to support cross platform tests would be to change the inline data variables to path components, then `Path.Join(...)` these in the test body, rather than hard-coding `/`

Reply:
If you want to open a new pull request with that method I wouldn't mind, I'm not quite sure what you're explaining though. That kind of seems like reimplementing the method in the test here.

Reply:
main goal is to get this working again so everything doesn't stay broken though haha